store/tikv/region_cache.go

```diff
@@ -297,6 +297,8 @@ func (c *RegionCache) Close() {
 
 // asyncCheckAndResolveLoop with
 func (c *RegionCache) asyncCheckAndResolveLoop() {
+	ticker := time.NewTicker(time.Minute)
```

Comment:
Make it configurable.

Reply:
Do you mean expose it in `configuration` or just provide a param in `asyncCheckAndResolveLoop`

Reply:
Expose it to users.

Reply:
updated.